gitpod-io/gitpod
# [ws-manager-mk2] Support workspace snapshots

## Description
Adds support for snapshotting workspaces

## Related Issue(s)
n.a.

## How to test
- Make sure the node is labeled, `kubectl label node fo-mk2-snapshots gitpod.io/experimental=true`
- Open workspace in preview environment
- Select "Share workspace snapshot" option and take snapshot
- After a while a snapshot should have been created

## Release Notes

```release-note
None
```
## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
